src/ui/czi-list.css

```diff
@@ -36,11 +36,15 @@
 }
 
 .ProseMirror li > p {
-  zoom: 1;
-  display: block;
+  /**
+   * Use `flow-root` to enforce Block formatting context so that the sibling
+   * element after the counter won't overlap.
+   */
+  display: flow-root;
```

Comment:
this isn't supported in IE, Edge, or Safari. Can you test those to check the resulting behavior is acceptable if we're supporting those?

Reply:
`flow-root` is [supported in chrome, Firefox, and Edge](https://caniuse.com/#feat=flow-root). and we can use `[column-span: all](https://developer.mozilla.org/en-US/docs/Web/Guide/CSS/Block_formatting_context)` for Safari and IE.

---

Below are the screenshots of different browsers tested.

### Before Fix

![image](https://user-images.githubusercontent.com/1504439/57411754-5d463180-71a3-11e9-8488-64a99c9f6113.png)

---

### After Fix

**chrome**

![image](https://user-images.githubusercontent.com/1504439/57411651-093b4d00-71a3-11e9-8db7-ce4ed8d9280d.png)

**Edge Latest**

![image](https://user-images.githubusercontent.com/1504439/57411680-207a3a80-71a3-11e9-9092-230ad081fdbf.png)

**Edge 15**

![image](https://user-images.githubusercontent.com/1504439/57411979-0d1b9f00-71a4-11e9-833c-e822d1d7d568.png)


**Firefox**

![image](https://user-images.githubusercontent.com/1504439/57411695-2ff98380-71a3-11e9-91f9-a469bb75c5c2.png)

**Safari**

![image](https://user-images.githubusercontent.com/1504439/57411721-4273bd00-71a3-11e9-807b-55fd9b8d0a58.png)